packages/itwin/quantity-formatting/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@itwin/quantity-formatting-react",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "React components and utilities for quantity formatting",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/viewer-components-react/tree/master/packages/itwin/quantity-formatting"
+  },
+  "type": "module",
+  "types": "./lib/quantity-formatting-react.d.ts",
+  "exports": {
+    ".": {
+      "types": "./lib/quantity-formatting-react.d.ts",
+      "default": "./lib/quantity-formatting-react.js"
+    }
+  },
+  "scripts": {
+    "build": "npm run copy:public && tsc 1>&2 && npm run copy:css",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "copy:css": "cpx \"./src/**/*.*css\" ./lib",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "clean": "rimraf lib",
+    "rebuild": "npm run clean && npm run build",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "lint:table": "eslint ./**/*.{ts,tsx} 1>&2 -f ./node_modules/@itwin/eslint-plugin/dist/formatters/no-internal-summary-with-table.js",
+    "test": "vitest",
+    "cover": "vitest run --coverage --silent"
+  },
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "iTwin",
+    "Quantity",
+    "Formatting",
+    "Units",
+    "iTwin.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "peerDependencies": {
+    "@itwin/core-bentley": "^5.1.1",
+    "@itwin/core-common": "^5.1.1",
+    "@itwin/core-frontend": "^5.1.1",
+    "@itwin/core-quantity": "^5.1.1",
+    "@itwin/ecschema-metadata": "^5.1.1",
+    "@itwin/itwinui-react": "^3.16.0",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
+  },
+  "devDependencies": {
+    "@itwin/build-tools": "^5.1.1",
+    "@itwin/core-bentley": "^5.1.1",
+    "@itwin/core-common": "^5.1.1",
+    "@itwin/core-frontend": "^5.1.1",
```

Comment:
make sure peers of core-frontend are defined as dev deps

Reply:
i've defined them all as devDeps now